packages/web/src/pages/VaultEditor/vaultEditorService.ts

```diff
@@ -140,33 +145,62 @@ export async function sendEditionToGovApproval(editSessionId: string): Promise<I
-      { ...vaultContractInfo, functionName: "maxBounty" }, // maxBounty
-      { ...vaultContractInfo, functionName: "asset" }, // asset
-      { ...vaultContractInfo, functionName: "decimals" }, // tokenDecimals
-    ],
+    contracts:
+      vault.version === "v2"
+        ? ([
+            { ...vaultContractInfo, functionName: "committee" }, // Committee multisig address
+            { ...vaultContractInfo, functionName: "committeeCheckedIn" }, // Is committee checked in
+            { ...registryContractInfo, functionName: "isVaultVisible", args: [vault.id as `0x${string}`] }, // Is registered
+            { ...vaultContractInfo, functionName: "totalAssets" }, // Deposited amount
+            { ...vaultContractInfo, functionName: "bountySplit" }, // bountySplit
+            { ...vaultContractInfo, functionName: "getBountyHackerHATVested" }, // hatsRewardSplit
+            { ...vaultContractInfo, functionName: "getBountyGovernanceHAT" }, // hatsGovernanceSplit
+            { ...vaultContractInfo, functionName: "maxBounty" }, // maxBounty
+            { ...vaultContractInfo, functionName: "asset" }, // asset
+            { ...vaultContractInfo, functionName: "decimals" }, // tokenDecimals
+          ] as any)
+        : ([
+            { ...claimsManagerContractInfo, functionName: "committee" }, // Committee multisig address
+            { ...claimsManagerContractInfo, functionName: "committeeCheckedIn" }, // Is committee checked in
+            { ...registryContractInfo, functionName: "isVaultVisible", args: [vault.id as `0x${string}`] }, // Is registered
+            { ...vaultContractInfo, functionName: "totalAssets" }, // Deposited amount
+            { ...claimsManagerContractInfo, functionName: "bountySplit" }, // bountySplit
+            { ...claimsManagerContractInfo, functionName: "getBountyHackerHATVested" }, // hatsRewardSplit
+            { ...claimsManagerContractInfo, functionName: "getBountyGovernanceHAT" }, // hatsGovernanceSplit
+            { ...claimsManagerContractInfo, functionName: "maxBounty" }, // maxBounty
+            { ...vaultContractInfo, functionName: "asset" }, // asset
+            { ...vaultContractInfo, functionName: "decimals" }, // tokenDecimals
+            { ...claimsManagerContractInfo, functionName: "getArbitrator" }, // arbitrator
+            { ...claimsManagerContractInfo, functionName: "arbitratorCanChangeBounty" }, // arbitratorCanChangeBounty
+            { ...claimsManagerContractInfo, functionName: "arbitratorCanChangeBeneficiary" }, // arbitratorCanChangeBeneficiary
+            { ...claimsManagerContractInfo, functionName: "arbitratorCanSubmitClaims" }, // arbitratorCanSubmitClaims
+            { ...claimsManagerContractInfo, functionName: "isTokenLockRevocable" }, // isTokenLockRevocable
+          ] as any),
   }) as Promise<any[]>;
 
   const promisesData = await Promise.all([descriptionHashPromise, allContractCallsPromises]);
```

Comment:
promise all gave us issues. maybe worth to fix it

Reply:
in this case makes sense, because if someone fail is because something is wrong...